docs/testing/webdriverio/04-visual-testing.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Visual Testing
+sidebar_label: Visual Testing
+description: Visual Testing
+slug: /testing/webdriverio/visual-testing
+---
+
+# Mocking
+
+WebdriverIO supports [visual testing capabilities](https://webdriver.io/docs/visual-testing) out of the box through a plugin called [`@wdio/visual-service`](https://www.npmjs.com/package/@wdio/visual-service). It uses [ResembleJS](https://github.com/Huddle/Resemble.js) under the hood to do pixel perfect comparisons.
+
+## Adding Visual Testing to your Setup
+
+If you don't have a WebdriverIO project set up yet, please take a look at the set up instructions we provide on the [WebdriverIO Overview](./01-overview.md) page.
+
+Once you are set up, add the visual plugin to your project via:
+
+```bash npm2yarn
+npm install --save-dev @wdio/visual-service
+```
+
+Next, add the service to your services list in your `wdio.conf.ts`:
```

Comment:
Can we add a sentence or two here that explains what a sentence is? We do this a bit in the comments in the following code block, but I think it might flow a bit better if we define this term as a part of starting to use it